examples/tutorials/sklean_models.ipynb

```diff
@@ -0,0 +1,325 @@
+{
```

Comment:
<p>There is no need in <code>before</code> pipeline here. Put everything into the main pipeline.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>